packages/main/src/Tokenizer.js

```diff
@@ -181,7 +185,7 @@ class Tokenizer extends UI5Element {
 	}
 
 	get showNMore() {
-		return !this.expanded && this.showMore && this.overflownTokens.length;
+		return !this.expanded && this.showMore && this.overflownTokens.length && !this._hideNMoreText;
```

Comment:
You already set `expanded` to `true` so it should work out of the box I think, no need for changes here?